main.py

```diff
@@ -17,6 +17,29 @@
 
 PLAYER_STR_FORMAT = '{rank:2}) {name:{name_pad}} ({points:{points_pad}}) {stars}* ({star_time})\n'
 
+def insertion_sort(data_array):
+    
+    for index in range(1, len(data_array)):
+        current_data = data_array[index]
+        position = index
+        
+        while position > 0 and data_array[position - 1][1] > current_data[1]:
+            
+            data_array[position] = data_array[position - 1]
+            position = position - 1
+        
+        data_array[position] = current_data
+        
+    return data_array
```

Comment:
Ummm.  Just realised the only reason I did it was because I needed to sort it on the timestamps but I can do that using .sort() and a helper function.  Thanks!